### PR TITLE
chessboard creation

### PR DESCRIPTION
**Pull request by sanzencode**
No styling yet but the chessboard created.

### CODE REVIEW

File: app/views/games/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div id='chessboard'>
+<% ('a'..'h').each do |letter| %>
+	<div class='board_row'> 
+	<% (1..8).each do |number| %>
+		<%= content_tag :div, 'asdf', id: "#{letter}#{number}", class: 'chessboard_space' %>
```
Comment (thisguycodes) [inline app/views/games/new.html.erb]: What is the significance of 'asdf' here? What could it be instead?

_No significance, just place holder text_ - Sandra

File: app/views/games/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div id='chessboard'>
+<% ('a'..'h').each do |letter| %>
+	<div class='board_row'> 
+	<% (1..8).each do |number| %>
+		<%= content_tag :div, 'asdf', id: "#{letter}#{number}", class: 'chessboard_space' %>
+	<% end %>
+	</div>
+<% end %>
+</div>
```
Comment (thisguycodes) [inline app/views/games/new.html.erb]: Missing ending line break. In your editor this should look like the last line in your file is blank. There's a way to set your editor to automatically make sure of this.

**Comment by thisguycodes** [comment]
This looks fine, but I don't see where you're creating the used `chessboard_space` style. And could you add a screenshot of what this looks like to the PR description?

**Comment by justinrjoseph** [comment]
Seb, Sandra included the styling in app/assets/stylesheets/games.scss

Here's a screenshot of the board. Sorry if the quality is sub-par.
image

**Comment by sebtn** [comment]
@sanzencode @justinrjoseph, I just updated the gist according to your desing https://gist.github.com/37bd1ccc4fe57b27399c.git

**Comment by sanzencode** [comment]
I will adjust the chessboard to be numbers and not letters.

**Comment by justinrjoseph** [comment]
Since there was so much code fragmentation for the tasks this week, and this PR has become rather discombobulated...it might be best for us to ditch this PR. I think it would be helpful if I pushed a new branch and create a PR for drawing the pieces without all the problems present here. Agreeable?

**Pull request closed without merging**